.github/workflows/main.yml

```diff
@@ -14,6 +14,7 @@ jobs:
   lint:
     name: Lint
     runs-on: ubuntu-latest
+    timeout-minutes: 3
```

Comment:
I think the issue only happens on `test` and `test-create-ponder` right, because it's from `vitest`? Can probably remove from `lint` and `build` if so.